# jaguar arm motor command message from DrRobot Robot.
float32 posX		# arm tip end x position
float32 posY	        # arm tip end y position
float32 tipAngle  #tip angle(related with ground), unit:radian


